cirras-underwriting-war/cirras-underwriting-war/src/main/angular/src/app/store/verified-yield/verified-yield.effects.ts

```diff
@@ -104,134 +107,132 @@ export class VerifiedYieldEffects {
-
-//                     return [                                                                         
-//                       LoadDopYieldContractSuccess(typedAction.componentId, newdopYieldContract)                             
-//                     ]
-//                   }),
-//                   catchError(error =>{
-//                       return of(LoadDopYieldContractError(typedAction.componentId, convertToErrorState(error, "New DOP Yield "))) 
-//                   }
-//               ),
-//               )   
-//         }                  
-//     )
-// ))
-
-// // TODO:
-// updateDopYield: Observable<Action> = createEffect (() => this.actions.pipe(
-//     ofType(UPDATE_DOP),
-//     withLatestFrom(this.store),
-//     debounceTime(500),
-//     switchMap(
-//         ([action, store]) => {
-//             let typedAction = <DopYieldContractAction>action;
-//             let requestId = `WFDME${UUID.UUID().toUpperCase()}`.replace(/-/g, "");
-//             let authToken = this.tokenService.getOauthToken();
-//             let payload = <DopYieldContract>typedAction.payload.dopYieldContract;
-
-//             const dopYieldContract = typedAction.payload.dopYieldContract 
-//             const body: DopYieldContractRsrc = convertToDopYieldContractRsrc(dopYieldContract)
+              return this.CirrasUnderwritingAPIService.addANewVerifiedYieldContract(
+                body,
+                requestId,
+                REST_VERSION, 
+                "no-cache",  
+                "no-cache",  
+                `Bearer ${authToken}`,
+                "response")
+              .pipe(
+                  concatMap((response: any) => {
+
+                    displaySaveSuccessSnackbar(this.snackbarService, "Verified Yield ");
+
+                    let newVerifiedYieldContract: VerifiedYieldContract = convertToVerifiedYieldContract(response.body, response.headers ? response.headers.get("ETag") : null) 
+
+                    return [                                                                         
+                      LoadVerifiedYieldContractSuccess(typedAction.componentId, newVerifiedYieldContract)                             
+                    ]
+                  }),
+                  catchError(error =>{
+                     return of(LoadVerifiedYieldContractError(typedAction.componentId, convertToErrorState(error, "New Verified Yield "))) 
+                  }
+              ),
+              )   
+        }                  
+    )
+))
+
+updateDopYield: Observable<Action> = createEffect (() => this.actions.pipe(
+    ofType(UPDATE_VERIFIED_YIELD),
+    withLatestFrom(this.store),
+    debounceTime(500),
+    switchMap(
+        ([action, store]) => {
+            let typedAction = <VerifiedYieldContractAction>action;
+            let requestId = `WFDME${UUID.UUID().toUpperCase()}`.replace(/-/g, "");
```

Comment:
I noticed that we're using WFDME prefix for the requestId. Is that on purpose?

Reply:
No. It's a mistake I have to change it.